images/manageiq-base/Dockerfile

```diff
@@ -27,8 +30,7 @@ LABEL name="manageiq-base" \
       io.k8s.description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
       io.openshift.tags="ManageIQ,miq,manageiq"
 
-RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm && \
```

Comment:
Wonder if we could contact https://github.com/nodesource/distributions about getting ppc64le support for node builds.

I would rather open an issue over there first rather than completely change how we build and install node.

Reply:
I will give it a try